runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/SimpleStreamingWordCountTest.java

```diff
@@ -34,7 +36,9 @@
 import org.apache.beam.sdk.transforms.windowing.FixedWindows;
 import org.apache.beam.sdk.transforms.windowing.Window;
 import org.apache.beam.sdk.values.PCollection;
+
```

Comment:
It's not enforced by the style check (`mvn clean verify` was happy) so I figured it's a matter of taste. Is it not?